src/main/java/com/homework/lesson15/dao/CompanyDao.java

```diff
@@ -0,0 +1,75 @@
+package com.homework.lesson15.dao;
+
+import com.homework.lesson15.entities.Companies;
+import lombok.extern.log4j.Log4j;
+
+import javax.persistence.EntityManager;
+
+@Log4j
+public class CompanyDao {
```

Comment:
better use interface CompanyDao with needed methods and class CompanyDaoImpl that is implemented this interface